### PR TITLE
putting plot of most recent data against forecast on website

### PR DESCRIPTION
**Pull request by emchristensen**
but see issue 110

### CODE REVIEW

File: report.Rmd [hunk 1]
```diff
@@ -1,4 +1,40 @@
 ---
+title: "Report - September 2017"
+---
+Here's the most recent data (blue) compared to the forecast performed right before data collection (black, with error bars)
+```{r, echo=FALSE, message=FALSE, warning=FALSE}
+source("forecast_tools.R")
+
```
Comment (sdtaylor) [inline report.Rmd]: There needs to be a `library(portalr)` here

File: report.Rmd [hunk 2]
```diff
@@ -1,4 +1,40 @@
...
+joined_data[joined_data$species=='total','period'] = period_code
+
+sp_predict = plot_species_forecast(sp_predictions)  
+sp_predict = sp_predict + geom_point(data = joined_data, mapping = aes(x = abundance, y = species),
+             color = "blue")
+plot(sp_predict)
+```
+
+
+---
```
Comment (sdtaylor) [inline report.Rmd]: can you remove the old stuff on here (everything under the June report)? It's erroring out on me now anyway.

**Comment by emchristensen** [comment]
removed old code.  I had left it there because I thought it might be cool to keep a history of these, but you're right, the way the code is written the plots would change as the data changes anyway

**Comment by sdtaylor** [comment]
We'll have to save old reports some other way.

**Pull request merged**